.eslintrc.js

```diff
@@ -26,6 +26,8 @@ module.exports = {
 		'/projects/amd-loader/src/demo/modules',
 		'/projects/js-themes-toolkit/packages/liferay-theme-tasks/lib/r2',
 		'/projects/js-themes-toolkit/qa',
+		'/projects/js-toolkit/packages/js-api/**/*.d.ts',
+		'/projects/js-toolkit/packages/js-api/**/*.js',
```

Comment:
why ignore? Shouldn't the generated js and d.ts only be built when published?

Reply:
Yeah, the problem is that locally you will usually have them generated and the linter complains before sending a PR if you run it, for example. So, to make our lives easier...